Add Aqua Cloud Native Security Platform, Datadog Agent and Self-hosted Runner files (#113)
odidev/ecosystem-dashboard-for-arm: content/commercial_packages/Self-hosted_Runner.md

---
name: Self-hosted Runner
vendor: CircleCI
category: DevOps

> description: CircleCI’s self-hosted runner enables you to use your own infrastructure for running jobs.

Author: ```suggestion
description: CircleCI’s self-hosted runner enables users to use their own infrastructure for running jobs.
```

product_url: https://circleci.com/docs/runner-overview/
works_on_arm: true
release_date_on_arm: 17/11/2020

optional_info:
    homepage_url: https://circleci.com/docs/runner-overview/
    support_caveats:
    alternative_options:
    getting_started_resources:
        arm_content: 
        vendor_announcement: https://circleci.com/blog/self-hosted-runners-add-arm-support-to-circleci/
        official_docs: https://circleci.com/docs/runner-overview/#getting-started

optional_hidden_info:

>     other_info: The CircleCI milestones page https://circleci.com/milestones/ confirms the release date for the ARM platform.

Author: ```suggestion
    other_info: The CircleCI [milestones page](https://circleci.com/milestones/) confirms the release date for the ARM platform.
```
